tools/ogstools/msh2vtu.xml

```diff
@@ -0,0 +1,97 @@
+
+<tool id="msh2vtu" name="ogstools msh2vtu" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
+    <description>
+        Convert msh to vtu/vtk.
```

Comment:
I would change it from "Convert msh to vtu/vtk" to "with msh2vt".
This info is already given in the long description

Reply:
I would keep it as it is. My preference is to have the original tool name first. 

`Convert msh to UnstructuredGrid (vtk).`

Also please remove the `.` in the end